spring-boot-admin-samples/pom.xml

```diff
@@ -35,10 +35,11 @@
 
     <modules>
         <module>spring-boot-admin-sample-custom-ui</module>
-        <module>spring-boot-admin-sample-servlet</module>
         <module>spring-boot-admin-sample-reactive</module>
         <module>spring-boot-admin-sample-war</module>
         <module>spring-boot-admin-sample-hazelcast</module>
+        <module>spring-boot-admin-sample-oauth2</module>
+        <module>spring-boot-admin-sample-oauth2-client</module>
```

Comment:
We were recently discussing if it is really useful to add more samples here. It is increasing build-time and makes git operations slower.

Maybe it would be better to put them in https://github.com/codecentric/spring-boot-admin-runtime-playground and advertise this repo more. I will discuss that with the rest of the team.